build-system/externs/amp.extern.js

```diff
@@ -241,11 +270,20 @@ window.AMP.toggleExperiment;
 window.AMP.setLogLevel;
 window.AMP.setTickFunction;
 window.AMP.viewer;
-window.AMP.viewport = {};
+window.AMP.viewport;
```

Comment:
Are these statements declaring everything here as `?Object`?

Reply:
I have no idea what this defaults to, I've been assuming `?`